### ISSUE TITLE
Doesn't work on haxe 3.2.1

### ISSUE THREAD

**Issue by kevinresol** [ExternKit/extern-js#2]
Fails with `src/js/extern/Extern.hx:10: characters 15-21 : Unexpected extern`

The only reason I can think of is that the keyword `extern` is not allowed as package name...

**Comment by fantoine** [comment]
It seems that this keyword has been allowed in haxe 3.3. Since I work with nightly builds I didn't noticed that.
Maybe it should be renamed to something like `externkit` to keep the compatibility.

**Comment by kevinresol** [comment]
Agreed. There are [other problems](https://github.com/HaxeFoundation/haxe/issues/5267#issuecomment-221251194) as well.

I suggest putting the whole extern kit libraries under `externkit`, like: 
`package externkit.js;`
`package externkit.js.redis;`

**Comment by fantoine** [comment]
I also know about these issues.
I used `@:noPackageRestrict` to avoid the 'forbidden js package' it's a 'private' meta.

In a first hand, changing all packages should solve lots of issues.
But in an other hand, `externkit` looks very long to type to me and externs look too much "integrated" in a library.
Maybe we could just get rid of the `js` namespace, but we may get conflicts with some other libraries.

Some ideas:

``` haxe
import externkit.js.aws.cognito.Cognito;
// or
import ekt.js.aws.cognito.Cognito;
// or
import aws.js.cognito.Cognito;
// or
import externkit.aws.cognito.Cognito;
// or
import extkit.aws.cognito.Cognito;
// or
import aws.cognito.Cognito;
```

Personally, I prefer the last case. What's your opinion?

**Comment by kevinresol** [comment]
I prefer fullnames (unless really common terms like aws), so ekt, extkit are ruled out
I like the the last one but I would vote for the first one because it is much more explicity
